### DISCUSSION TITLE
part-ii/2021-117/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#372] [Question comments]
Part II 2021 Principles of Statistics: Paper 1, Section II, J 

https://questions.tripos.org/part-ii/2021-117/

**Comment by ArthurConmy** [comment]
<details>
<summary>b)</summary>

I get Fisher information $ 1 + (n-1)(1-\gamma) $ so tensorises only if $\gamma = 0$. Then lower bound is just the reciprocal of this ... ?

</details>
